packages/amplication-client/src/Application/NewApplication.tsx

```diff
@@ -103,30 +108,35 @@ const NewApplication = () => {
         isInitialValid={false}
       >
         {(formik) => {
+          const handlers = {
+            SUBMIT: formik.submitForm,
+          };
           return (
             <Form>
```

Comment:
Since this code repeats, maybe it's better to create a component that wraps Form and HotKeys together.

Reply:
#573